sources/packages/web/src/services/CASSupplierService.ts

```diff
@@ -0,0 +1,38 @@
+import ApiClient from "@/services/http/ApiClient";
+import {
+  AddCASSupplierAPIInDTO,
+  CASSupplierInfoAPIOutDTO,
+} from "@/services/http/dto";
+
+export class CASSupplierService {
+  // Share Instance
+  private static instance: CASSupplierService;
+
+  public static get shared(): CASSupplierService {
+    return this.instance || (this.instance = new this());
+  }
+
+  /**
+   * Gets supplier info by student id.
+   * @param studentId student id.
+   * @returns CAS supplier info.
+   */
+  async getSupplierInfoByStudentId(
+    studentId: number,
+  ): Promise<CASSupplierInfoAPIOutDTO[]> {
```

Comment:
I believe the preferred way for the DTOs would be to have the array as a property.
I tried to find similar examples and found somewhere a DTO was created with a single array property.